lib/account/models/profile.dart

```diff
@@ -51,6 +51,9 @@ class Profile extends Model {
 
   bool get isCurrentUser => id == supabaseManager.currentProfile?.id;
 
+  static int get maxAge => DateTime(1900).year;
+  static int get minAge => DateTime.now().subtract(const Duration(days: 365 * 12)).year;
```

Comment:
a year is not an age, that makes no sense

Just use `static const int minAge = const Duration(days: 365 * 12)`. As for the oldest possible age, let's just use 120 or something?

Reply:
I changed it to just const ints. Do we really want 12 as the minimum Age. Or should we go for 16 or 18

Reply:
Oh sorry I meant `Duration`, I don't think `int` adds any valuable information?

Also please use the constant in `EditBirthDate` too, that was the whole point imo.

I think 12 is fine (why should they not be allowed?), but maybe we should not allow young people to create drives? Does not really matter too much to me though.